src/components/SettingsSection/SettingsSection.vue

```diff
@@ -48,7 +48,9 @@ This component is to be used in the settings section of nextcloud.
 				:href="docUrl"
 				class="settings-section__info"
 				role=""
-				:title="docTitleTranslated" />
+				:title="docTitleTranslated">
+				<Information :size="24" decorative />
```

Comment:
Hm, depending on where i hover, i get either the right text or just the icon-aria-label
![grafik](https://user-images.githubusercontent.com/47433654/113359403-fe19d900-9347-11eb-9953-cafddc40fe39.png)
![grafik](https://user-images.githubusercontent.com/47433654/113359559-4b964600-9348-11eb-9822-c736910efb3a.png)

Reply:
Ah, I guess I need to set the title to empty text for all icons :grin:

Reply:
Done :rocket: